### PR TITLE
Replaced Bootstrap tour by IntroJs in Labyrinth Activity

### PR DESCRIPTION
**Pull request by ABHINAV-JHA-27**
Migrated Labyrinth Activity from Bootstrap Tour to IntroJs https://github.com/llaske/sugarizer/issues/1090

### CODE REVIEW

**Comment by ABHINAV-JHA-27** [comment]
Hey @llaske , can you have a look

File: activities/LabyrinthJS.activity/index.html [hunk 1]
```diff
@@ -2,65 +2,66 @@
 <html>
 
 <head>
-<meta charset="utf-8" />
-<title>LabyrinthJS Activity</title>
-<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="prefetch" type="application/l10n" href="locale.ini" />
-<link rel="stylesheet" media="not screen and (device-width: 1200px) and (device-height: 900px)"
-  href="lib/sugar-web/graphics/css/sugar-96dpi.css">
+  <meta charset="utf-8" />
```
Comment (llaske) [inline activities/LabyrinthJS.activity/index.html:5]: Please do not change indentation style. Only updated lines should be changed in your PR.
It's probably due to options in your IDE.

Comment (ABHINAV-JHA-27) [reply activities/LabyrinthJS.activity/index.html:5]: Sorry my bad, will be fixing it.

**Comment by llaske** [comment]
@ABHINAV-JHA-27 index.html is still reformated

**Pull request closed without merging**